tools/qemu_dev_start

```diff
@@ -62,9 +65,12 @@ print_help()
 	printf '\t%s\n' "<rootfs>: Path to a x86-64-generic-rootfs.tar.gz"
```

Comment:
The usage line (--help) has to be updated also.  
We have been using argbash https://argbash.io/generate to maintain the options of the script (and helper line, etc), can you please use it? (you can copy from the line 0 until # ARGBASH_GO() into the argbash textbox and then copy the result back to the script).

Reply:
thanks! was unaware of that. rebased and force-pushed my branch, using argbash